_integration-schemas/3plcentral/locations.md

```diff
@@ -0,0 +1,139 @@
+---
+tap: "3plcentral"
+version: "1.0"
+
+name: "locations"
+doc-link: "http://api.3plcentral.com/rels/"
+singer-schema: "https://github.com/singer-io/tap-3plcentral/blob/master/tap_3plcentral/schemas/locations.json"
+description: "This table contains information about locations."
```

Comment:
But what _type_ of locations? Looking at the API docs, it looks like it's inventory locations.